docs/topics/wasm/wasm-debugging.md

```diff
@@ -18,66 +19,68 @@ Create a project using the Kotlin Multiplatform wizard:
    >
    {type="tip"}
 
-3. Select the **Web** option. Make sure that no other options are selected.
+3. Select the **Web** option. You don't need to select any other platform options for this tutorial.
 4. Click the **Download** button and unpack the resulting archive.
 
-![Kotlin Multiplatform wizard](wasm-compose-wizard.png){width=600}
+   ![Kotlin Multiplatform wizard](wasm-compose-wizard.png){width=600}
 
-## Open the project in IntelliJ IDEA
+## Open the project in Fleet
 
-1. Download and install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/).
-2. On the Welcome screen of IntelliJ IDEA, click **Open** or select **File | Open** in the menu bar.
-3. Navigate to the unpacked "WasmDemo" folder and click **Open**.
+1. Download and install the latest version of [Fleet](https://www.jetbrains.com/fleet/).
+2. On the Welcome screen of Fleet, click **Open File or Folder** or select **File | Open** in the menu bar.
+3. Navigate to the unpacked "WasmDemo" folder and click **Open**. 
 
-## Run the application
+## Run and debug your application in Fleet
 
-1. In IntelliJ IDEA, open the **Gradle** tool window by selecting **View** | **Tool Windows** | **Gradle**.
+1. Open the `Greeting.kt` code file from the `composeApp/src/wasmJsMain/kotlin` directory.
+2. Click the line numbers to set breakpoints on the code you want to inspect.
+   In this example, we set the breakpoint on the line **5**. The number turns into a red circle.
 
-   > You need at least Java 11 as your Gradle JVM for the tasks to load successfully.
-   >
-   {type="note"}
+   ![Breakpoints](wasm-fleet-breakpoints.png){width=600}
 
-2. In **composeApp** | **Tasks** | **kotlin browser**, select and run the **wasmJsBrowserRun** task.
+3. Open the `main.kt` code file from the `composeApp/src/wasmJsMain/kotlin` directory.
+4. Click the **Run** button on the `main()` function line and select **Debug 'main'**.
```

Comment:
Should we mention that the `Smart Mode` should be turned on, and the user should wait for the indexing to finish? For some users, it can be not obvious why there is no `Run` gutter icon until the indexing finished